website/pages/en/sunrise.mdx

```diff
@@ -1,233 +1,79 @@
-
-### Will subgraphs need to be re-indexed again?
-
-No, rest assured that your subgraph will not need to be re-indexed when it is upgraded to The Graph Network. Subgraphs will be immediately available for querying, powered by the upgrade Indexer, which uses the same infrastructure as the hosted service. If your subgraph is indexing a network that is eligible for indexing rewards, you can add signal to attract indexers. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
-
-### I’m experiencing indexing issues. What should I do?
-
-Rest assured that network Indexers are prepared to provide support during this upgrade. If you experience issues with any of your subgraph queries, please reach out to support@thegraph.zendesk.com
-
-### Why is my subgraph being published to Arbitrum, is it indexing a different network?
-
-The Graph Network was originally deployed on mainnet Ethereum but moved to Arbitrum One to reduce gas costs for all users. As such any new subgraphs are published to The Graph Network on Arbitrum so that they can be supported by Indexers. Arbitrum is the network that subgraphs are published to, but subgraphs can index any of the [supported networks](https://thegraph.com/docs/en/developing/supported-networks/)
-
-### How can I get started querying subgraphs on The Graph Network?
-
-You can explore available subgraphs on [Graph Explorer](https://thegraph.com/explorer). [Learn more about querying subgraphs on The Graph](/querying/querying-the-graph).
+The Graph Network was initially deployed on the mainnet Ethereum but was later moved to Arbitrum One in order to lower gas costs for all users. As a result, any new subgraphs were published on Arbitrum to The Graph Network so that Indexers could support them. Arbitrum is the network that subgraphs are published to, but subgraphs can index any of the [supported networks](https://thegraph.com/docs/en/developing/supported-networks/)
 
 ## About the Upgrade Indexer
 
-### What is the upgrade Indexer?
-
-The upgrade Indexer is designed to improve the experience of upgrading subgraphs from the hosted service to The Graph Network and to support new versions of existing subgraphs that have not yet been indexed.
-
-The upgrade Indexer aims to bootstrap chains that don't have indexing rewards yet on The Graph Network and to serve as a fallback for new subgraph versions. The goal is to ensure that an Indexer is available to serve queries as quickly as possible after a subgraph is published.
+> The upgrade Indexer is currently active
 
-### What chains does the upgrade Indexer support?
+The upgrade Indexer improved the experience of upgrading subgraphs from the hosted service to The Graph Network and supported new versions of existing subgraphs that had not yet been indexed.
```

Comment:
```suggestion
The upgrade Indexer was deployed to improve the experience of upgrading subgraphs from the hosted service to The Graph Network and support new versions of existing subgraphs that had not yet been indexed.
```
(not 100% sure about the term "deployed" here, I just don't think "improve" or "support" should be in the past tense)

Reply:
"implemented"